docs/configuration.md

```diff
@@ -568,6 +568,13 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.ui.enabled</code></td>
+  <td>true</td>
+  <td>
+    Whether to run web ui service inside spark application.
```

Comment:
Oh, hm, it's not documented? I don't think it's secret, so OK. Let's say "... run the web UI for the Spark application"